### PR TITLE
changes to reduce build time

### PR DESCRIPTION
**Pull request by paraglade**
changes `gzip` to using multi threaded gzip compression; adjustments to zstd compression aggressiveness.

### CODE REVIEW

File: scripts/package-images [hunk 1]
```diff
@@ -11,8 +11,8 @@ for FILE in build/images*.txt; do
     BASE=$(basename ${FILE} .txt)
     TARFILE=build/images/${PROG}-${BASE}.tar
     cp -f ${FILE} dist/artifacts/${PROG}-${BASE}.${PLATFORM}.txt
-    zstd -T0 -16 -f --long=25 --no-progress ${TARFILE} -o dist/artifacts/${PROG}-${BASE}.${PLATFORM}.tar.zst
-    gzip -v -c ${TARFILE} > dist/artifacts/${PROG}-${BASE}.${PLATFORM}.tar.gz
+    zstd -T0 -8 -f --exclude-compressed --no-progress ${TARFILE} -o dist/artifacts/${PROG}-${BASE}.${PLATFORM}.tar.zst
+    pigz -v -c ${TARFILE} > dist/artifacts/${PROG}-${BASE}.${PLATFORM}.tar.gz
```
Comment (brandond) [inline scripts/package-images:15]: I'm OK with the change to pigz, but the zstd settings (`-16 --long=25`) need to be retained in order balance compression factor against the amount of memory used by RKE2 when decompressing the resulting tarball. `--exclude-compressed` should be safe to add, but I would like to see a comparison of the archive sizes before and after.

**Review by brandond — CHANGES_REQUESTED** [review]

File: scripts/package-images (hunk 1 above)
Comment (paraglade) [reply scripts/package-images:15]: From the testing I did comparing various options to zstd I was seeing around 2% difference in compression from a `-16` to `-8` on the compression level which I would suspect would have very little impact on RKE2. However this little change has resulted in over 10 minute faster build times. I am thinking any improvement to build times would be welcome here especially something as simple as this.

Comment (brandond) [reply scripts/package-images:15]: What about the `--long=25` bit, what's the impact of that? It needs to be in sync with
https://github.com/rancher/wharfie/blob/main/pkg/tarfile/tarfile.go#L28-L33

Comment (paraglade) [reply scripts/package-images]: Thanks for bringing this to my attention. I have put this option back in and will review the build for this change.

**Pull request closed without merging**